components/Layout/Layout.tsx

```diff
@@ -1,7 +1,10 @@
 /* library package */
 import { useEffect } from 'react'
 import { ToastContainer, toast } from 'react-toastify'
-import { withBrand, Newsletter } from '@sirclo/nexus'
+import {
+  withBrand,
+  Newsletter,
+  useI18n} from '@sirclo/nexus'
```

Comment:
coba dibikin gini mungkin ya raff
```suggestion
  useI18n
  } from '@sirclo/nexus'
```